src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -98,14 +99,14 @@ class OpenChatGlobalAction extends Action2 {
 		}
 		if (opts?.previousRequests?.length && chatWidget.viewModel) {
 			for (const { request, response } of opts.previousRequests) {
-				chatService.addCompleteRequest(chatWidget.viewModel.sessionId, request, undefined, 0, { message: response });
+				chatService.addCompleteRequest(chatWidget.viewModel.sessionId, request, variables ? { variables } : undefined, 0, { message: response });
 			}
 		}
 		if (opts?.query) {
 			if (opts.isPartialQuery) {
-				chatWidget.setInput(opts.query);
+				chatWidget.setInput(opts.query, variables);
```

Comment:
Maybe it's easier to call `chatWidget.attachmentModel.addContext` here than add this plumbing?